manager-bundle/tests/Fixtures/HttpKernel/WithMixedNamespace/src/EventListener/InvalidListener.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Foo\EventListener;
+
+class InvalidListener
```

Comment:
This listener causes an issue that makes the `--prefer-lowest` tests fail: https://github.com/contao/contao/pull/1165/checks?check_run_id=393652779#step:6:40

<img width="1059" alt="" src="https://user-images.githubusercontent.com/1192057/72563648-6947ce80-38ae-11ea-92bb-c02144077052.png">

Reply:
Fixed in e669b8284c2f58c9f76ffdf9919ed8260bfaef9c.